extension/src/state_aggregate.rs

```diff
@@ -279,11 +292,13 @@ mod tests {
                 None,
             );
             assert_eq!(
-                31536120000000,
+                "8760:02:00",
```

Comment:
Presumably we wouldn't want to call out to postgres for every timestamptz calculation, but it is worth considering that this a bit different from the postgres result:
```
postgres=# select ('2020-12-31 00:02:00+00'::timestamptz - '2020-01-01 00:00:00+00'::timestamptz)::interval;
     interval      
-------------------
 365 days 00:02:00
(1 row)
```

Reply:
It took me a while to figure out how and where postgresql is doing this, but I did find it, and I've documented it here, and I'm calling the same function.

Thanks!